### PR TITLE
Update: README.md

### PR DESCRIPTION
**Pull request by shivam-sharma7**
Update: README.md

### CODE REVIEW

**Comment by shivam-sharma7** [comment]
review/ @jesperpedersen

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -41,18 +41,18 @@ cd distribution/target/distribution/
 
 We publish the image at [quay.io/hyperfoil/hyperfoil](https://quay.io/repository/hyperfoil/hyperfoil?tab=tags).
 
-## Contributing
+## 🧑‍💻 Contributing
 
-Contributions to `Hyperfoil` are managed on [GitHub.com](https://github.com/Hyperfoil/Hyperfoil/)
+Contributions to `Horreum` Please check our [CONTRIBUTING.md](https://github.com/Hyperfoil/Horreum/blob/master/CONTRIBUTING.md)
```
[the remark this answers is not in the document]
Comment (shivam-sharma7) [reply README.md:46]: I thought that we don't need to create another and then I have linked Horreum CONTRIBUTING.md file.

[the remark this answers is not in the document]
Comment (jesperpedersen) [reply README.md:46]: Every project needs to have their own

[the remark this answers is not in the document]
Comment (jesperpedersen) [reply README.md:46]: Nothing has changed - you need to include the file, and update the URL

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ We publish the image at [quay.io/hyperfoil/hyperfoil](https://quay.io/repository
 
 ## 🧑‍💻 Contributing
 
-Contributions to `Horreum` Please check our [CONTRIBUTING.md](https://github.com/Hyperfoil/Horreum/blob/master/CONTRIBUTING.md)
+Contributions to `Hyperfoil` Please check our [CONTRIBUTING.md](https://github.com/Hyperfoil/Horreum/blob/master/CONTRIBUTING.md)
```
Comment (jesperpedersen) [inline README.md:46]: Replace Horreum

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (shivam-sharma7) [reply README.md:46]: Replace already

File: README.md [hunk 3]
```diff
@@ -41,18 +41,18 @@ cd distribution/target/distribution/
 
 We publish the image at [quay.io/hyperfoil/hyperfoil](https://quay.io/repository/hyperfoil/hyperfoil?tab=tags).
 
-## Contributing
+## 🧑‍💻 Contributing
 
-Contributions to `Hyperfoil` are managed on [GitHub.com](https://github.com/Hyperfoil/Hyperfoil/)
+Contributions to `Hyperfoil` Please check our [CONTRIBUTING.md](https://github.com/Hyperfoil/Horreum/blob/master/CONTRIBUTING.md)
```
Comment (jesperpedersen) [inline README.md:46]: It is still Horreum - everything needs to be separate

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

**Pull request closed without merging**